src/token/CustomERC721.sol

```diff
@@ -162,16 +175,40 @@ contract CustomERC721 is NonReentrant, ContractMetadata, InitializableLazyMint,
   }
 
   /**
-   * @notice Used internally to initialize the contract instead of through a constructor
-   * @dev This function is called by the deployer/factory when creating a contract
+   * @notice Initialize the lazy minting for the contract
+   * @dev This function also synchronizes the metadata with the prepended tokenID 
    */
   function initLazyMint() external override onlyOwner returns (uint256 chainPrepend) {
-    require(!_isLazyMintInitialized(), "HOLOGRAPH: lazy mint already initialized");
+    if(_isLazyMintInitialized()) revert LazyMint_AlreadyInitialized();
```

Comment:
prettier

Reply:
Running `pnpm run prettier:fix` doesn't change anything, I don't understand this one 😄

Reply:
it's missing space between `if(`